src/patterns/validation/index.md.njk

```diff
@@ -0,0 +1,92 @@
+---
+title: Recover from errors (validation)
+description: Check the answers users give to make sure they’re valid - and if there’s an error, tell them what's wrong and how to fix it
+section: Patterns
+theme: Help users to…
+aliases: 
+backlog_issue_id: 
+layout: layout-pane.njk
+---
+
+Check the information the user gives you to make sure it’s valid. If there's a validation error, tell the user what's gone wrong and how to fix it. Turn off HTML5 validation.
+
+## When to use this pattern
+
+Validation should be a last resort. Minimise your chances of needing to show an error message by:
+
++ [making sure your questions are well designed](https://www.gov.uk/service-manual/design/designing-good-questions)
++ accepting information in different formats, as long as it’s not ambiguous (for example, you should accept postcodes with or without spaces - or names with non-alphabetical characters like accents)
```

Comment:
> alphabetical 

I think these are diacritics. Aren't letters like èáö still alphabetical?

Reply:
Could include some examples of diacritic characters.

Reply:
Thanks @edwardhorsford  - have clarified that validation should accept diacritics as well as non-alphabetical characters